core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -93,8 +93,6 @@ describe('About Page', () => {
 
     expect(siteAnalyticsService.registerCreateLessonButtonEvent)
       .toHaveBeenCalledWith();
-    expect(windowRef.nativeWindow.location.href).toBe(
```

Comment:
Just wondering why we are dropping these other expectations?

Reply:
Because we are no longer using those about-page functions to navigate through pages